card-1/css/style.scss

```
@@ -0,0 +1,2 @@
+@import "global";
+@import "container";
```

```suggestion
@import "container";

```
Prazan red na kraju svakog fajla je obavezan